workloads/hyper-scale/common.sh

```diff
@@ -51,6 +53,19 @@ setup(){
     rosa verify permissions
 }
 
+pre_flight_checks(){
+    echo "Pre flight checks started"
+    export MULTI_AZ=$(rosa describe cluster -c krvoora-router -o json | jq -r [.multi_az] | jq -r .[])
+    export COMPUTE_NODE=$(rosa describe cluster -c krvoora-router -o json | jq -r [.nodes.compute] | jq -r .[])
+
+    if [ "${MULTI_AZ}" == "true" ] && [ "${COMPUTE_NODE}"/3 == 0]; then
```

Comment:
I think you want % and not / when checking if the compute nodes are divisible by 3

Reply:
Thanks for the catch, updated